ROX-21316: Add info for compliance operator (#91470)
openshift/openshift-docs: operating/compliance-operator-rhacs.adoc

[role="_abstract"]
You can configure {product-title-short} to use the Compliance Operator for compliance reporting and remediation with {ocp} clusters. Results from the Compliance Operator are reported in the {product-title-short} Compliance Dashboard.

> While {product-title-short} comes with several default compliance profiles, these only scan the user workloads. The Compliance Operator adds compliance profiles suitable for the rest of the {ocp} infrastructure; therefore, installing the Compliance Operator provides a complete environment scanning solution.

Reviewer: ```suggestion
The default compliance profiles in {product-title-short} only scan the user workloads. The Compliance Operator adds compliance profiles suitable for the rest of the {ocp} infrastructure. Therefore, installing the Compliance Operator provides a complete environment scanning solution.
```
